libraries/serviceescalation.rb

```diff
@@ -126,16 +134,17 @@ def escalation_options=(arg)
     # rubocop:disable MethodLength
     def config_options
       {
-        'service_description'   => 'service_description',
-        'contacts_list'         => 'contacts',
-        'contact_groups_list'   => 'contact_groups',
-        'escalation_period'     => 'escalation_period',
-        'host_name_list'        => 'host_name',
-        'hostgroup_name_list'   => 'hostgroup_name',
-        'escalation_options'    => 'escalation_options',
-        'first_notification'    => 'first_notification',
-        'last_notification'     => 'last_notification',
-        'notification_interval' => 'notification_interval'
+        'service_description'    => nil,
```

Comment:
For me, this causes the error: Error: Service escalation host name or description is NULL